openmc/stats/univariate.py

```diff
@@ -256,6 +257,59 @@ def integral(self):
         """
         return np.sum(self.p)
 
+    def clip(self, tolerance: float = 1e-6, inplace: bool = False) -> Discrete:
+        r"""Remove low-importance points from discrete distribution.
+
+        Given a probability mass function :math:`p(x)` with :math:`\{x_1, x_2,
+        x_3, \dots\}` the possible values of the random variable with
+        corresponding probabilities :math:`\{p_1, p_2, p_3, \dots\}`, this
+        function will remove any low-importance points such that :math:`\sum_i
+        x_i p_i` is preserved to within some threshold.
+
+        .. versionadded:: 0.13.4
+
+        Parameters
+        ----------
+        tolerance : float
+            Maximum fraction of integral of the product of `x` and `p` that will
```

Comment:
since this is only for `Discrete` objects should we describe this as the maximum fraction of :math:`\sum_i x_i p_i` that will be discarded to use the same terminology as the docstring?